kafo.gemspec

```diff
@@ -23,11 +23,14 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'simplecov', '< 0.9'
   spec.add_development_dependency 'ci_reporter', '~> 1.9.0'
 
+  spec.add_development_dependency 'puppet'
+  # this is required if puppet is greater than 4.0
+  # spec.add_development_dependency 'puppet-strings'
```

Comment:
Should this be in the Gemfile so it's easier to enable? I assume you're suggesting devs would have to uncomment it.